privatelink/azure/dns-endpoints.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+
+#
+# dns-endpoints.sh
+#
+# Output zone records to correctly map to zonal endpoints for Confluent Cloud.
+#
+# Example:
+#
+#   % ./dns-endpoints.sh my-resource-group private-endpoint-1 private-endpoint-2 private-endpoint-3
```

Comment:
I guess we need an example for Single Zone and maybe some fixes for Single Zone.

Reply:
Handled that in the new commit. PTAL.